cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -0,0 +1,109 @@
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.CpuCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.PotentialNwOutGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.CpuUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.TopicReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderBytesInDistributionGoal";
+
+    /**
+     * A list of case insensitive goals that Cruise Control supports in the order of priority.
+     * The high priority goals will be executed first.
+     */
+    private static final String DEFAULT_GOAL_PRIORITY = "com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.CpuCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.PotentialNwOutGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.CpuUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.TopicReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderBytesInDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.kafkaassigner.KafkaAssignerDiskUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.kafkaassigner.KafkaAssignerEvenRackAwareGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.PreferredLeaderElectionGoal";
```

Comment:
Why there are two different goals lists which are also slightly different?

Reply:
Yes, this could very easily go out of sync and cause weird bugs. Why not have the list of defaults in the default order?

Reply:
`DEFAULT_GOALS` is the for the default list of goals to be used and `DEFAULT_GOAL_PRIORITY` is a list ranking of all goals by priority. I guess I can do as @tomncooper suggests and list the goals in `DEFAULT_GOALS` by priority and then set `DEFAULT_GOAL_PRIORITY` = `DEFAULT_GOALS`. It's worth noting that unless we include all the goal in the `DEFAULT_GOAL` list, some goals will not be ranked/prioritized